src/lib/eina/eina_module.c

```diff
@@ -27,13 +27,11 @@
 #include <libgen.h>
 #include <unistd.h>
 
-#if defined HAVE_DLOPEN && ! defined _WIN32
+#if defined HAVE_DLOPEN
 # include <dlfcn.h>
 #endif
 
-#ifdef _WIN32
-# include <evil_private.h>
-#endif
```

Comment:
I have no idea, actually, but removing it didn't do anything different by what I could see.

Reply:
Ok, talked to vtorri. This is here because of "dl*" things, and since this PR defines `dl*` things in `dlfcn.h`, I couldn't see differences in compile errors.